docs/serving/autoscaling/concurrency.md

```diff
@@ -74,13 +74,14 @@ spec:
 
 ### Hard limit
 
-The hard limit has no global setting and can only be specified [per revision](./autoscaling-concepts.md).
-This particular per-revision setting is not an annotation; it is present on the revision's spec itself as `containerConcurrency`.
+The hard limit is always specified [per revision](./autoscaling-concepts.md) via the `containerConcurrency` field on the revision spec. This particular per-revision setting is not an annotation; it is present on the revision's spec itself as `containerConcurrency`.
+
+There is no global setting for the hard limit in the autoscaling config map, but a default value can be set for the Revision's `containerConcurrency` field in `config-defaults.yaml`.
 
 * The default value is `0`, meaning that there is no limit on the number of requests that are allowed to flow into the revision.
 * A value greater than `0` specifies the exact number of requests that are allowed to flow to the replica at any one time.
 
-* **Global key:** No global key.
+* **Global key:** No global key (but a default can be set for the per-revision spec key in `config-defaults.yaml`).
```

Comment:
Hmm, let's document the global key in the other configmap then and add an example. We can just add a remark that it's in a different CM.

You might want to add that it's not in the AS configMap because it touches more aspects (namely queuing and buffering which are somewhat concerns of networking) which is also why this isn't just yet another autoscaling annotation but an integral part of the spec IIRC

Reply:
okie, gave that a go